src/fontra/views/editor/edit-tools-pen.js

```diff
@@ -416,7 +416,20 @@ function setupExistingAnchorPoint(context, path, point, shiftConstrain) {
   );
 }
 
-function insertAnchorPoint(context, path, point, shiftConstrain) {
+function insertAnchorPoint(context, path, point, shiftKey) {
+  if (shiftKey) {
+    // Shift-constrain the point 0/45/90 degrees
+    if (context.createContour === false) {
```

Comment:
Please combine this in the `if (shiftKey)` condition, like this:
```js
  if (shiftKey && !context.createContour) {
```
(Generally: don't compare with `false` or `true` directly.)